src/ducks/apps/components/AppInstallation.jsx

```diff
@@ -16,16 +17,27 @@ import Checkbox from 'cozy-ui/react/Checkbox'
 
 import { getTranslatedManifestProperty } from 'lib/helpers'
 import { hasPendingUpdate } from 'ducks/apps/appStatus'
+import CTS from 'config/constants'
 
 import { APP_TYPE, getAppBySlug, installAppFromRegistry } from 'ducks/apps'
 
+const shouldSkipPermissions = app =>
+  flags('skip-low-permissions') && app.label <= CTS.default.authorizedLabelLimit
+
+const hasInstallation = app => !app.installed || hasPendingUpdate(app)
+
 export class AppInstallation extends Component {
   constructor(props) {
     super(props)
+    const { app } = props
     this.state = { isTermsAccepted: false }
+    if (shouldSkipPermissions(app) && hasInstallation(app)) {
+      this.installApp()
+    }
+    this.installApp = this.installApp.bind(this)
```

Comment:
I would have maybe binded this method before calling `this.installApp()` right above. It feels strange to call it and then bind it.